mk/kind.mk

```diff
@@ -151,10 +152,12 @@ kind/deploy/metrics-server:
 
 .PHONY: kind/deploy/example-app
 kind/deploy/example-app:
+	@KUBECONFIG=$(KIND_KUBECONFIG) kubectl apply -n $(EXAMPLE_NAMESPACE) -f dev/examples/k8s/meshes/no-passthrough.yaml
+	@KUBECONFIG=$(KIND_KUBECONFIG) kubectl apply -n $(EXAMPLE_NAMESPACE) -f dev/examples/k8s/external-services/httpbin.yaml
+	@KUBECONFIG=$(KIND_KUBECONFIG) kubectl apply -n $(EXAMPLE_NAMESPACE) -f dev/examples/k8s/external-services/mockbin.yaml
```

Comment:
was that only for local testing or we want to have this in master?

Reply:
I use this make target extensively for various tests. Having a default mesh with mTLS and no pasthrough + a couple of external services can only speed up my development. I'd like to have this upstream.